app/browser/menu.js

```diff
@@ -622,17 +634,28 @@ const doAction = (state, action) => {
         const frame = frameStateUtil.getFrameByTabId(state, action.tabId)
         if (frame) {
           currentLocation = frame.location
-          setMenuItemChecked(state, locale.translation('bookmarkPage'), isCurrentLocationBookmarked(state))
+          setMenuItemAttribute(state, locale.translation('bookmarkPage'), 'checked', isCurrentLocationBookmarked(state))
+        }
+        break
+      }
+    case appConstants.APP_WINDOW_CLOSED:
+    case appConstants.APP_WINDOW_CREATED:
+      {
+        const windowCount = BrowserWindow.getAllWindows().filter((w) => w.isFocused()).length
+        if (action.actionType === 'app-window-closed' && windowCount === 0) {
```

Comment:
could you use the constant name here instead?
`if (action.actionType === appConstants.APP_WINDOW_CLOSED && windowCount === 0) {`